components/03-modules/modal/modal--header-footer.hbs

```diff
@@ -0,0 +1,42 @@
+{{#if trigger }}
+    {{ render '@modal-trigger' modalTrigger }}
+{{/if}}
+<div role="dialog"
+     aria-hidden="true"
+     id="{{ modal.id }}"
+     data-modal="{{ modal.id }}"
+     class="modal {{ modal.class }}"
+     tabindex="-1"
+     {{{ modal.attributes }}}
+ >
+    <div role="document"
+         class="modal__content {{ modalContent.class }}"
+         {{{ modalContent.attribtues }}}
+         tabindex="0"
```

Comment:
remove tabindex from this element, it's not necessary

Reply:
done

Reply:
hmmm. Inside dialog we should focus first focusable element, but you're right - we can set this focus on the content area, to not close the popup immediately. Sorry, we can re-add this, but only to this element, not to role="dialog" :)